torch/_dynamo/config.py

```diff
@@ -60,6 +60,9 @@
 # don't specialize on shapes and strides and put shape ops in graph
 dynamic_shapes = os.environ.get("TORCHDYNAMO_DYNAMIC_SHAPES") == "1"
 
+# user provided specs for dynamic args
+dynamic_args = None
```

Comment:
Don't put this in the config, shape env seems better.

Reply:
Moved to ShapeEnv.